resources/index.scss

```diff
@@ -469,7 +469,14 @@ table.simple {
 
 }
 
-// Width base classes
+// Display EMail Template rich text
+.force-normalize-fonts {
+    h1, h2, h3, h4, h5 {
+        font-size: $body-font-size;
+    }
+}
+
+// Width base classes (TODO: Framework integration)
```

Comment:
ping todo

Reply:
Will remove the comment when I update the framework package. I am not sure yet what we will need to include in the framework